Modules/Files/test/Files.API.Tests.Performance.EndToEnd/run-all.ps1

```diff
@@ -0,0 +1,17 @@
+Param(
+    [parameter(Mandatory)] $baseUrl,
+    [parameter(Mandatory)] $size
+)
+
+Write-Host "Running files performance tests..."
+
+npm install
+node ".\node_modules\webpack\bin\webpack.js"
+
+Get-ChildItem -Path "./dist" -Filter *.test.js |
+
+Foreach-Object {
+    k6 run -e HOST=$baseUrl -e SIZE=$size -e CLIENT_SECRET=test $_.FullName
```

Comment:
Via chat I told you that the way to run composite tests, as described in the first comment of the GitHub issue you found (https://github.com/grafana/k6/issues/1342), looks promising. Why didn't you implement that instead of running one by one?